test/run-test.ps1

```diff
@@ -20,9 +20,9 @@ else {
     $optionalDockerBuildArgs = "--no-cache"
 }
 
-$dockerRepo = "microsoft/dotnet-nightly"
 $dirSeparator = [IO.Path]::DirectorySeparatorChar
 $repoRoot = Split-Path -Parent $PSScriptRoot
+$dockerRepo = (Get-Content "${repoRoot}${dirSeparator}manifest.json" | ConvertFrom-Json).DockerRepo
```

Comment:
changed to 
$manifestPath = [IO.Path]::combine(${repoRoot}, "manifest.json")
$dockerRepo = (Get-Content $manifestPath | ConvertFrom-Json).DockerRepo